### ISSUE TITLE
Update Mapping to Use Sources Other Than Stock

### ISSUE THREAD

**Issue by sidrus** [TeleIO/Telemachus-1#32]
Actually, here is the specific post in the thread, with the *beta* link that includes complete JNSQ maps:
https://forum.kerbalspaceprogram.com/index.php?/topic/181688-kerbal-maps-is-back-sort-of/&do=findComment&comment=3756346

And if any code needs to changed in Telemachus to use JNSQ *or ANY other pack from there*,
maybe changes could/should be made to future-proof and make it easier for Telemachus to use different packs from there? (like either a UI or config/settings file for Telemachus to switch between packs, rather than having something that would need to changed, being hard-coded?) vOv
i know there was uncomfirmed comments/discussion made about maybe adding some other popular planet packs to the KM catalog, in the future... vOv

_Originally posted by @StoneBlue in https://github.com/TeleIO/Telemachus-1/issues/19#issuecomment-606849579_

**Comment by Row-Bear** [comment]
One place I've found reference to the Kerbal Maps site is in
https://github.com/TeleIO/Telemachus-1/blob/develop/WebPages/WebPages/src/js/leafletksp.js

**Comment by StoneBlue** [comment]
Yeah, thats where I found the reference, and swapped in the URL for the test server (w/JNSQ) maps

**Comment by sidrus** [comment]
I have a first draft of this up on development.  The only maps implemented so far are the Stock/JNSQ Kerbin and Stock Mun.  I will get the rest implemented over the next couple of days.

**Comment by sidrus** [comment]
So with @Row-Bear 's latest PR, we have all the bodies for both JNSQ and stock, and we have an extensible way to add more sources as they are available in Kerbal Maps.  I see some improvements we can make in the UI to help declutter it a bit, but I think we've met the intent of this ticket, @StoneBlue ?

**Comment by SolAstrius** [comment]
This was implemented — stock and JNSQ bodies (Kerbin, Mun, and the rest) are available via the layer switcher in `map.html`, with an extensible structure for adding further sources. Closing as done. If there are specific bodies or tile sources still missing, feel free to open a new focused issue.
